docs/usage/files.md

```diff
@@ -520,3 +520,59 @@ client.files.listRepresentations(
 ```
 
 [get-representations]: https://opensource.box.com/box-ios-sdk/Classes/FilesModule.html#/s:6BoxSDK11FilesModuleC19listRepresentations6fileId18representationHint10completionySS_AA018FileRepresentationJ0OSgys6ResultOySayAA0lM0VGAA0A8SDKErrorCGctF
+
+Create Zip
```

Comment:
Per comment above, let's make Create Zip private, so this section can be removed.